templates/menu_link/menu_link.component.inc

```diff
@@ -6,6 +6,32 @@
  */
 
 /**
+ * Implements hook_preprocess_HOOK().
+ */
+function europa_preprocess_menu_link__main_menu(&$variables, $hook) {
+  $link = $variables['element']['#original_link'];
+  $variables['attributes_array']['class'][] = 'ecl-navigation-menu__item';
+  $variables['output']['#options']['attributes']['class'][] = 'ecl-navigation-menu__link';
+  if ($link['in_active_trail']) {
+    if ($link['depth'] == 1) {
+      $variables['attributes_array']['class'][] = 'ecl-navigation-menu__item--active';
+    }
+    else {
+      $variables['output']['#options']['attributes']['class'][] = 'ecl-navigation-menu__link--active';
+    }
+  }
+  // If the menu item has a submenu and has a depth of 1,
+  // render it as a button.
+  if (!empty($variables['sub_menu']) && $link['depth'] == 1) {
+    $variables['output']['#options']['attributes']['id'][] = 'nav-menu-expandable-label-' . $link['mlid'];
+    $variables['output']['#options']['attributes']['aria-controls'][] = 'nav-menu-expandable-group-' . $link['mlid'];
+    $variables['output']['#options']['attributes']['aria-expanded'][] = 'false';
+    $variables['output']['#options']['attributes']['aria-haspopup'][] = 'true';
+    $variables['output'] = '<button ' . drupal_attributes($variables['output']['#options']['attributes']) . '>' . $variables['output']['#text'] . '</button>';
```

Comment:
As we are in a preprocess function, I would replace the HTML by the render array.

```
$variables['output'] = array(
  '#type' => 'html_tag',
  '#tag' => 'button',
  '...',
);
```